### DISCUSSION TITLE
I love my badges, I feel like i'm special! Are you too?

### DISCUSSION THREAD

**Discussion by jadehamel** [community/community#30469] [Other Feature Feedback, Questions, & Ideas]
Since I got badges, strangely I want to get more so bad! Does it give you some addictive patterns too?

**Comment by ArtemMyzrov** [comment]
Yeah this is a great story ! Cool )

**Comment by Daizygod** [comment]
it`s like achivments in videogames, i have 2 platinum achivments in ps5 and because of this, I feel myself better when I see this in my profile

**Comment by drippy-cat** [comment]
Hi jadehamel,

Were glad to hear that you enjoy earning badges on GitHub! To submit an idea to GitHub, try contacting them or posting a new discussion!

Happy coding!

-DC

**Comment by antonmedv** [comment]
I love badges too ❤️ so I created custom badges: https://github.com/my-badges/my-badges

You can define your own badges too. 

Check this out: 
<img width="1287" alt="Screenshot 2023-10-05 at 22 20 12" src="https://github.com/community/community/assets/141232/b0ab63bd-6b67-43b2-8230-589bda624b58">
